internal/restic/node_xattr.go

```diff
@@ -16,6 +17,10 @@ import (
 // Getxattr retrieves extended attribute data associated with path.
 func Getxattr(path, name string) ([]byte, error) {
 	b, e := xattr.Get(path, name)
+	if err, ok := e.(*xattr.Error); ok && err.Err == syscall.ENODATA {
```

Comment:
Please merge the if condition to check for `ENODATA` and `ENOTSUP`.

Reply:
thanks for the feedback. 
I have updated the commit and add a issue template